ui/components/component-library/avatar-account/avatar-account.constants.js

```diff
@@ -1,3 +1,5 @@
+import { SIZES } from '../../../helpers/constants/design-system';
+
 export const DIAMETERS = {
```

Comment:
Can we update these const names to use the prefix `AVATAR_ACCOUNT` e.g. `AVATAR_ACCOUNT_TYPES` and `AVATAR_ACCOUNT_DIAMETERS`